build/builder/Dockerfile

```diff
@@ -128,8 +128,8 @@ RUN git clone --depth 1 https://github.com/tpoechtrager/osxcross.git \
 
 # Compiling Go from source requires a Go toolchain to bootstrap.
```

Comment:
We built go from source in the past so we could apply the parallel-cgo patch. Is there any reason we continue to do so instead of using the binary releases? (no need to change in this PR, of course)

Reply:
No reason that I can think of!